packages/InteractiveIcon/svgs/CaretDown.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { forwardRef } from 'react'
+
+import LimitedInteractiveIcon, { StyledLimitedInteractiveIconSVG } from '../LimitedInteractiveIcon'
+
+const CaretDown = forwardRef((props, ref) => (
+  <LimitedInteractiveIcon {...props} ref={ref}>
+    <StyledLimitedInteractiveIconSVG
+      animationDirection="down"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+    >
+      <svg>
```

Comment:
`StyledLimitedInteractiveIconSVG` already renders an svg, this svg is not needed.